config/detekt/config.yml

```diff
@@ -313,7 +313,7 @@ naming:
   ConstructorParameterNaming:
     active: true
     parameterPattern: '[a-z][A-Za-z0-9]*'
-    privateParameterPattern: '[a-z][A-Za-z0-9]*'
+    privateParameterPattern: '_?[a-z][A-Za-z0-9]*'
```

Comment:
Can you please revert that? The `_something` is a legacy pattern and we don't want it in our new codebase